docs/databags.md

```diff
@@ -33,6 +33,15 @@ The License Hash is part of a data bag item encrypted with [Chef Vault](https://
 
 * `[A Decriptive File-Name]` - Corresponding License (XML) contents. (There can be many of these) Remember to change newlines to `\n` to conform to proper JSON format.
 
+License Pool Hash
+------------
+This hash is part of a plaintext data bag item that defines multiple license pools. Indexers can be assigned to a specific pool and license quota can be set for each pool by configuring this databag. 
```

Comment:
Some additional details about the format for the sizes would be useful as well. e.g. 
> Sizes can be specified in KB, MB, GB or TB and must include the unit (e.g. '400GB').  

Since it applies to most of the attributes in this hash maybe just state it once up here in the general description?